src/models/thelio-major-r5-n3/repairs.md

```diff
@@ -0,0 +1,641 @@
+- The CPU cooler is an [Arctic Freezer 4U-M](https://www.arctic.de/us/Freezer-4U-M/ACFRE00133A).
+- The CPU uses an AMD sTR5 socket.
+
+**Tools required:** Cross-head (Phillips) screwdriver, AMD Threadripper torque tool or compatible 1.5 Nm (13.275 in-lb) torque 3.95mm six-point star (Torx 20) screwdriver, thermal paste  
+**Time estimate:** 45 minutes  
+**Difficulty:** High <span style="color:red;">●</span>
+
+### Steps to remove the CPU cooler/thermal paste:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case), [remove the side brace](#removing-the-side-brace), [remove the CPU duct](#removing-the-cpu-duct), [remove the GPU](#replacing-the-gpu), and [remove the heatsink-mounted CPU fans](#steps-to-replace-the-heatsink-mounted-cpu-fans).
+2. While holding the CPU cooler in place so it doesn't fall, loosen the screws to the left and right of the CPU cooler.
+    - The two screws on the left side use the outer holes.
+    - The two screws on the right side use the inner holes.
+
+![CPU cooler screws](./img/cpu-cooler-screws.webp)
+
+3. Lower the cooler away from the CPU.
+4. Using a paper towel, clean the existing thermal paste off of the heatsink and CPU.
+    - Due to the design of the heatsink, some thermal paste may remain between the heatsink pipes. Only the flat section of the pipes that makes contact with the CPU needs to be cleaned.
+    - You may also use a small amount of rubbing alcohol if the old paste is dried or difficult to remove.
+
+![Thermal paste removal](./img/thermal-paste-removal.webp)
+
+### Steps to replace the CPU:
+
+1. Place the computer on its side so the motherboard is facing up.
+2. Using the AMD Threadripper torque screwdriver (or a compatible T20 screwdriver), loosen the three CPU bracket Torx screws in reverse order of the printed numbers (starting with #3).
+    - The screws are held captive and will not come out of the bracket.
+
+![CPU Torx screws](./img/cpu-bracket-screws.webp)
+
+3. The CPU bracket cover will spring up when all screws are fully loosened.
+    - If the bracket cover only partially opens, you may need to loosen screws #2 and/or #3 more.
+
+![CPU bracket (open)](./img/cpu-bracket-open.webp)
+
+4. Using the blue corner tabs (visible above), flip the orange CPU carrier frame up and away from the CPU socket.
+
+![CPU carrier frame (raised)](./img/cpu-carrier-raised.webp)
+
+5. Slide the CPU carrier tray out of the CPU bracket.
+    - Be careful not to bend any of the gold pins on the CPU socket, and do not touch the gold pads on the CPU.
+
+![CPU (removed)](./img/cpu-removed.webp)
+
+6. Slide the new CPU's carrier tray into the CPU bracket.
+    - AMD Threadripper CPUs come with carrier trays, so removing the CPU from the tray is generally not necessary.
+7. Flip the CPU bracket back down onto the socket, gently pressing the blue tabs to seat the CPU carrier tray onto the motherboard.
+8. Flip the CPU bracket cover back down.
+9. Screw the three Torx screws back into place.
```

Comment:
I think the screw torque order (1-2-3) should be mentioned in the tightening directions